lib/stripe/subscriptions/subscription.ex

```diff
@@ -124,7 +124,7 @@ defmodule Stripe.Subscription do
                optional(:trial_from_plan) => boolean,
                optional(:trial_period_days) => non_neg_integer
              }
-  def create(%{customer: _, items: _} = params, opts \\ []) do
```

Comment:
THis ensures that the create action is passed with a customer and items.

Reply:
Do you think we need this?  It seems these two are required.

Reply:
ping!

Reply:
Oh sorry, I think it's necessary to provide it for the sake of keeping in line with the tutorial. If we'd like to figure out if the user has passed in a customer and items key, we can probably just use the `Map.has_key?(params, :customer)` and if it returns false, we throw an exception.